azurerm/internal/services/appconfiguration/app_configuration_resource.go

```diff
@@ -408,3 +447,30 @@ func flattenAppConfigurationAccessKey(input appconfiguration.APIKey) []interface
 		},
 	}
 }
+
+func expandAppConfigurationIdentity(d *schema.ResourceData) *appconfiguration.ResourceIdentity {
+	identities := d.Get("identity").([]interface{})
+	if len(identities) == 0 {
+		return &appconfiguration.ResourceIdentity{}
```

Comment:
Rather than just send an empty identity here, could we emphasize that we are expecting an identity of type `None` here?
```suggestion
		return &appconfiguration.ResourceIdentity{
		   Type: appconfiguration.None,
		}
```